### ISSUE TITLE
50 recipient limit doesn't apply to strings

### ISSUE THREAD

**Issue by daveliepmann** [danielcompton/clojure-postmark#13]
The 50-recipient limit doesn't seem to get applied when the `to` field is supplied as a comma-separated string instead of a vector. 

This check also seems to differ from the Postmark documentation, in that it only checks the `to` field:

>You can pass multiple recipient addresses in the To, Cc, and Bcc fields. Multiple addresses are comma separated and have a limit of 50 max recipients per message. This means that all receipients(To, Cc, and Bcc) are totaled and not allowed to exceed 50.

Additionally, it would be nice to clarify the predicate name while we're in there. Maybe something like `fifty-or-fewer-recipients?`?

**Comment by danielcompton** [comment]
Yep, all good catches. We'll want to check it's not too slow to count the recipients in the string, but that should be ok.
